src/test/scala/io/ergodex/core/sim/lqmining/simple/DepositBox.scala

```diff
@@ -20,9 +21,11 @@ final class DepositBox[F[_]: RuntimeState](
       // Context (declarations here are only for simulations):
       val PoolId: Coll[Byte]         = getConstant(1).get
       val RedeemerProp: Coll[Byte]   = getConstant(3).get
+      val RefundPk: Boolean          = getConstant(6).get
```

Comment:
This won't work, real constant is of type `ProveDlog` while we simplify it to `Boolean`. Coercion will fail.